tests/cryosms.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+
+from utils.test_modes import TestModes
+from utils.channel_access import ChannelAccess
+from utils.ioc_launcher import get_default_ioc_dir
+from utils.testing import get_running_lewis_and_ioc
+
+DEVICE_PREFIX = "CRYOSMS_01"
+EMULATOR_NAME = "cryogenic_sms"
+
+IOCS = [
+    {
+        "name": DEVICE_PREFIX,
+        "directory": get_default_ioc_dir("CRYOSMS"),
+        "emulator" : EMULATOR_NAME,
+    },
+]
+
+
+TEST_MODES = [TestModes.DEVSIM]
+
+
+class CryoSMSTests(unittest.TestCase):
+    def setUp(self):
+        self._lewis, self._ioc = get_running_lewis_and_ioc(EMULATOR_NAME, DEVICE_PREFIX)
+        self.ca = ChannelAccess(device_prefix=DEVICE_PREFIX, default_timeout=10)
+
+        self.ca.assert_that_pv_exists("DISABLE", timeout=30)
+
+    def test_GIVEN_outputmode_sp_correct_WHEN_outputmode_sp_written_to_THEN_outputmode_changes(self):
```

Comment:
This test is failing for me with the following error trace:

```
Running tests...
----------------------------------------------------------------------
F
======================================================================
FAIL [11.104s]: test_GIVEN_outputmode_sp_correct_WHEN_outputmode_sp_written_to_THEN_outputmode_changes (tests.cryosms.CryoSMSTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Instrument\Apps\EPICS\support\IocTestFramework\master\tests\cryosms.py", line 40, in test_GIVEN_outputmode_sp_correct_WHEN_outputmode_sp_written_to_THEN_outputmode_changes
    self.ca.assert_that_pv_is("OUTPUTMODE", new_outputmode, timeout=10)
  File "C:\Instrument\Apps\EPICS\support\IocTestFramework\master\utils\channel_access.py", line 259, in assert_that_pv_is
    pv, lambda val: val == expected_value, timeout=timeout, message=msg, pv_value_source=pv_value_source)
  File "C:\Instrument\Apps\EPICS\support\IocTestFramework\master\utils\channel_access.py", line 237, in assert_that_pv_value_causes_func_to_return_true
    raise AssertionError(err)
AssertionError: Expected PV, 'TE:NDW1799:CRYOSMS_01:OUTPUTMODE' to have value 'TESLA' (type: 'str').
Final PV value was 'AMPS' (type: 'str')

----------------------------------------------------------------------
Ran 1 test in 11.104s

FAILED (failures=1)
```

Note: I get the same error trace when testing IOC 2. When you have fixed this, please also check that IOC 2 is fixed.